### PR TITLE
Relay Command Add

### PR DESCRIPTION
**Pull request by codedeal**

### CODE REVIEW

File: DeviceControlApp/ViewModel/HomePageViewModel.cs [hunk 1]
```diff
@@ -8,8 +8,9 @@ namespace DeviceControlApp.ViewModel
 {
     public class HomePageViewModel
     {
-        public ICommand GoToNextCommand { get; private set; }  
       
+        public RelayCommand GoToNextCommand { get; private set; }
```
Comment (nachiappannk) [inline DeviceControlApp/ViewModel/HomePageViewModel.cs]: This should be ICommand. Only the implementation should be Relay Command type

File: DeviceControlApp/ViewModel/RelayCommand.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Windows.Input;
+
+namespace DeviceControlApp.ViewModel
+{
+    public class RelayCommand:ICommand
+    {
+        readonly Action<object> _execute;
+        readonly Predicate<object> _canExecute;
+        public RelayCommand(Action<object> execute, Predicate<object> canExecute)
```
Comment (nachiappannk) [inline DeviceControlApp/ViewModel/RelayCommand.cs]: public RelayCommand(Action exeute):this(exeute,null)
        {

        }
        public RelayCommand(Action execute, Predicate canExecute)
        {

        }

Comment (nachiappannk) [reply DeviceControlApp/ViewModel/RelayCommand.cs]: Add these constructors

Comment (nachiappannk) [reply DeviceControlApp/ViewModel/RelayCommand.cs]: _execute  = (obj) =>{ execute.Invoke(); }

File: DeviceControlApp/ViewModel/HomePageViewModel.cs (hunk 1 above)
Comment (codedeal) [reply DeviceControlApp/ViewModel/HomePageViewModel.cs]: Replaced new build with Relay command

File: DeviceControlApp/ViewModel/RelayCommand.cs (hunk 2 above)
Comment (codedeal) [reply DeviceControlApp/ViewModel/RelayCommand.cs]: Implemented above constructor

**Pull request merged**